app/src/main/java/com/sopt/now/LoginViewModel.kt

```diff
@@ -0,0 +1,40 @@
+package com.sopt.now
+
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.sopt.now.api.ServicePool
+import com.sopt.now.dto.LoginDto.RequestLogInDto
+import com.sopt.now.dto.LoginDto.ResponseLogInDto
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class LoginViewModel : ViewModel() {
+    val loginResult: MutableLiveData<Boolean> = MutableLiveData()
+    val errorMessage: MutableLiveData<String> = MutableLiveData()
+    val userId: MutableLiveData<String> = MutableLiveData()
```

Comment:
baking property 적용해주세요! 아마 다음주에 배울겁니다
[backing property](https://yoon-dailylife.tistory.com/45)

```suggestion
private val _loginResult = MutableLiveData<Boolean>()
val loginResult: LiveData<Boolean> get() = _loginResult

private val _errorMessage = MutableLiveData<String>()
val errorMessage: LiveData<String> get() = _errorMessage

private val _userId = MutableLiveData<String>()
val userId: LiveData<String> get() = _userId
```

값을 할당할때는 
```kotlin
_loginReult.postValue(true) // 혹은
_loginReult.setValue(true)
_userId.value = "" 
```

3가지 방법이 있는데 차이점을 알아보는 것도 좋습니다! (아마 다음주에..)